src/main/kotlin/com.bugsnag.android.gradle/BugsnagMultiPartUploadRequest.kt

```diff
@@ -37,10 +36,10 @@ class BugsnagMultiPartUploadRequest(
     private val overwrite: Boolean,
     private val endpoint: String,
     private val retryCount: Int,
-    timeoutDuration: Duration
+    okHttpClient: OkHttpClient
 ) {
 
-    private val bugsnagService = createService(timeoutDuration)
+    private val bugsnagService = createService(okHttpClient)
```

Comment:
Was very happy with how nicely this played with our previously set up overloads 😁